scripts/ts-utils/initApis.ts

```diff
@@ -0,0 +1,20 @@
+const { ApiPromise, WsProvider } = require("@polkadot/api");
+const { cryptoWaitReady } = require('@polkadot/util-crypto');
+import colors from "colors";
+//fetchApi is used to fetch data from the chain
+const wsFetchProvider = new WsProvider("wss://tee-staging.litentry.io:443");
```

Comment:
I still think `wss://tee-staging.litentry.io:443` should be a parameter.

Reply:
Fixed!Thank you for your suggestions. They will help improve my coding and engineering thinking in the future.